client/src/modules-data.json

```diff
@@ -0,0 +1,176 @@
+{
+  "modules": [
+    {
+      "id": 1,
+      "module_name": "Html-Css",
+      "topics": [
+        {
+          "topic_id": 1,
+          "topicName": "Fundamentals • Semantics",
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 2,
+          "topicName": "Forms • Structuring Data",
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 3,
+          "topicName": "Layout • Flex box • Grid",
+          "topicRef": "www.HTML-CSS.com"
+        },
+        {
+          "topic_id": 3,
```

Comment:
I guess you repeated topic 3 twice and missed topic 4. Can you fix that, please?

Reply:
I have fixed the topic 4. The issue has been resolved.